packages/path_provider/path_provider_linux/lib/path_provider_linux.dart

```diff
@@ -36,11 +36,11 @@ class PathProviderLinux extends PathProviderPlatform {
 
   @override
   Future<String> getApplicationDocumentsPath() {
-    return Future.value(xdg.getUserDirectory('DOCUMENTS').path);
+    return Future.value(xdg.getUserDirectory('DOCUMENTS')!.path);
```

Comment:
I am unsure if we can assume `xdg.getUserDirectory('DOCUMENTS')` is null only if the directory is incorrect thus forcing it wouldn't be an issue or not ?